tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/sequencefile/SequenceFileScanner.java

```diff
@@ -105,6 +105,11 @@ public void init() throws IOException {
     }
 
     String delim  = meta.getProperty(StorageConstants.SEQUENCEFILE_DELIMITER, StorageConstants.DEFAULT_FIELD_DELIMITER);
+    if (delim == null || delim.isEmpty()) {
+      delim = meta.getProperty(StorageConstants.TEXT_DELIMITER, StorageConstants.DEFAULT_FIELD_DELIMITER);
+    }
+    meta.getPropertySet().set(StorageConstants.TEXT_DELIMITER, delim);
```

Comment:
I think you can change to simplify code like this 
```
TableMeta tableMeta = meta.clone();
    if (!tableMeta.containsProperty(StorageConstants.TEXT_DELIMITER)) {
      tableMeta.putProperty(StorageConstants.TEXT_DELIMITER, tableMeta.getProperty(StorageConstants.SEQUENCEFILE_DELIMITER);
    }
```
also you should consider the null char and you should remove useless code